### ISSUE TITLE
Cannot find module after install package

### ISSUE THREAD

**Issue by jseerig** [forsti0506/a11y-sitechecker#564]
Hi there,
I just install the package witch yarn and as a 2. try with npm. If I call the script with ` a11y-sitechecker https://www.test.at --config=config.json -T=1000` for example, I get this error. 
What I have done wrong? What is missing? I hope you will help me

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/........./....../test-sitechecker/node_modules/a11y-sitechecker/bin/a11y-sitechecker_code' imported from /Users/JuliensMacBook/workspace/test-pov-sitechecker/node_modules/a11y-sitechecker/bin/a11y-sitechecker.js

**Comment by andres-leon** [comment]
I am getting the same error Here is the complete message I get:

`
node:internal/modules/esm/resolve:265
    throw new ERR_MODULE_NOT_FOUND(
          ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/usr/local/lib/node_modules/a11y-sitechecker/bin/a11y-sitechecker_code' imported from /usr/local/lib/node_modules/a11y-sitechecker/bin/a11y-sitechecker.js
    at finalizeResolution (node:internal/modules/esm/resolve:265:11)
    at moduleResolve (node:internal/modules/esm/resolve:933:10)
    at defaultResolve (node:internal/modules/esm/resolve:1157:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)
    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:227:38)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:87:39)
    at link (node:internal/modules/esm/module_job:86:36) {
  code: 'ERR_MODULE_NOT_FOUND',
  url: 'file:///usr/local/lib/node_modules/a11y-sitechecker/bin/a11y-sitechecker_code'
}

Node.js v20.14.0
`

Installing on Ubuntu 22.04.4 LTS.
